docs/loader_usage.md

```diff
@@ -0,0 +1,114 @@
+# Scylla Cluster Tests (SCT): Loaders and Stress Commands
+
+This document explains how to configure and use loaders and stress commands effectively in Scylla Cluster Tests (SCT). It primarily focuses on Longevity Tests (`longevity_test.LongevityTest.test_custom_time`), although the concepts apply similarly to other tests, which might include additional parameters (refer to specific test code/documentation for details).
+
+---
+
+## Overview of Loaders
+
+Loaders are specialized machines used to generate workloads on Scylla clusters. They run stress tools such as:
+
+- [cassandra-stress](https://github.com/scylladb/cassandra-stress/) - the most commonly used stress tool, simulating various workloads on Scylla with predefined columns count, row size and user profiles for custom ones.
+- [scylla-bench](https://github.com/scylladb/scylla-bench/) - a newer stress tool designed for Scylla, providing more advanced features especially for testing large partitions.
+- [gemini](https://github.com/scylladb/gemini/) - tool for data consistency validation across various schemas and mixed workloads (read, write, delete). Verifies data integrity against 'oracle' cluster.
+- [latte](https://github.com/scylladb/latte/) - advanced stress tool, mostly used for custom schemas (supports many data types and is very flexible). `rust` based.
+- [cql-stress](https://github.com/scylladb/cql-stress/) - `rust` based equivalent of `cassandra-stress` and `scylla-bench` - supports the same command line options.
```

Comment:
AFAIK, not `the same command line options`, but partial for now.
There is no 100% parity, right?

Reply:
close to 100% (for now I'm aware only about `extra_definitions` in user profiles, which is planned in the near future)

Reply:
Ok, we can keep it as-is because the API is really designed to be alike even if it is not 100% coverage.